modules/nw-metallb-bgppeer-cr.adoc

```diff
@@ -26,14 +26,20 @@ The fields for the BGP peer custom resource are described in the following table
 
 |`spec.myASN`
 |`integer`
-|Specifies the Autonomous System number for the local end of the BGP session.
+|Specifies the Autonomous System Number (ASN) for the local end of the BGP session.
 Specify the same value in all BGP peer custom resources that you add.
 The range is `0` to `4294967295`.
 
 |`spec.peerASN`
 |`integer`
-|Specifies the Autonomous System number for the remote end of the BGP session.
-The range is `0` to `4294967295`.
+|Specifies the ASN for the remote end of the BGP session.
+The range is `0` to `4294967295`. 
+If you use this field, you cannot specify a value in the `spec.dynamicASN` field.
+
+|`spec.dynamicASN`
+|`string`
+| Detects the ASN to use for the remote end of the session without explicitly setting it.
+Specify `internal` if the routers have the same ASN or `external` if the routers have different ASNs. If you use this field, you cannot specify a value in the `spec.peerASN` field.
```

Comment:
on second thought maybe instead of "the routers" we should use "the peer", since it seems we don't refer to the cluster nodes as "routers" anywhere else. wdyt?

Reply:
thanks for the review @oribon